### PR TITLE
add markdown view

### PR DESCRIPTION
**Pull request by go-oleg**
Used via `view markdown -field 'fieldToShow'`. If no `-field` specified, looks for a field other than `time` to show.

Fixes #66 .

If it can't find a field to show or a specified field is blank, don't show anything.

@mnibecker

### CODE REVIEW

**Comment by jadbox** [comment]
Neat!

File: test/views/markdown.spec.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+        view.el.innerHTML.should.contain('abc');
+    });
+
+    it('renders last point', function() {
+        var view = new MarkdownView();
+
+        view.consume([ { someField: 'abc' }, { someField: 'def' } ]);
+
+        view.el.innerHTML.should.contain('def');
```
Comment (mattnibs) [inline test/views/markdown.spec.js:32]: may be add `view.el.innerHTML.should.contain('abc');` as well here?

Comment (go-oleg) [inline test/views/markdown.spec.js:32]: It should not contain `abc` because it only shows the last point.

Comment (mattnibs) [inline test/views/markdown.spec.js:32]: right

**Comment by mattnibs** [comment]
+1

**Comment by go-oleg** [comment]
resolves #62

**Pull request merged**